R/background.R

```diff
@@ -58,7 +59,8 @@ rcmdcheck_process <- R6Class(
 
     initialize = function(path = ".", args = character(),
       libpath = .libPaths(), repos = getOption("repos"))
-      rcc_init(self, private, super, path, args, libpath, repos),
+      rcc_init(self, private, super, path, check_args = character(),
+               build_args = character(), libpath, repos),
```

Comment:
I find it mildly confusing if `check_args` and `build_args` are named but the following arguments aren't. Maybe just create two local variables?

The original `args` argument seems unused now, too.

Reply:
I forgot about the `args` in the `initialize()` call.
I moved the named args of `check_args()` and `build_args()` to the initialize function. Before I got an error due to an empty default. 

In summary it should be solved by moving the named args to the initialize function definition instead of the `rcc_init()` call. 

Thanks for checking :+1: